bundles/org.openhab.binding.bluetooth.bluegiga/src/main/java/org/openhab/binding/bluetooth/bluegiga/handler/BlueGigaBridgeHandler.java

```diff
@@ -152,12 +153,14 @@
     private final Map<Integer, BluetoothAddress> connections = new ConcurrentHashMap<>();
 
     private volatile boolean initComplete = false;
+    private volatile boolean initRequest = false;
```

Comment:
Is this boolean even necessary? Would just checking `serialHandler.get().isAlive()` suffice?

Reply:
Yes and done